documentation/src/developer_documentation/application_architecture.md

```diff
@@ -0,0 +1,16 @@
+# The application architecture of `forest` largely mirrors that of `lotus`:
+
+- There is a core
+  [`StateManager`](https://github.com/ChainSafe/forest/blob/v0.8.2/blockchain/state_manager/src/lib.rs),
+  which accepts:
+  - [RPC calls](https://github.com/ChainSafe/forest/blob/v0.8.2/node/rpc/src/lib.rs)
+  - [Filecoin peer state](https://github.com/ChainSafe/forest/blob/v0.8.2/blockchain/chain_sync/src/lib.rs)
+
+For more information, see the
+[lotus documentation](https://github.com/filecoin-project/lotus/blob/v1.23.0/documentation/en/architecture/architecture.md),
+including, where relevant, the
+[filecoin specification](https://spec.filecoin.io/).
+
+These also serve as a good introduction to the general domain, assuming a basic
+familiarity with blockchains. If you'd like to learn more about blockchains,
+[this 3b1b video is a good introduction](https://youtu.be/bBC-nXj3Ng4).
```

Comment:
I wouldn't put it on the `application architecture` page. As a user of this, I would expect to have it completely Forest-specific (and in this case Lotus, because we often translate their Go code to Rust).

On the other hand, we could create a separate page with educational materials; both general blockchain and Filecoin specific (e.g., https://github.com/filecoin-project/awesome-filecoin). What do you think?